packages/components/forms/src/FormControl/index.ts

```diff
@@ -0,0 +1 @@
+export { FormControl } from './CompoundFormControl';
```

Comment:
If we do not need to export the one not compound from './FormControl.tsx'

Can't we move all the logic of './CompoundFormControl' to './FormControl'?
IMO it would be less confusing, but that is just preference
what is everyone else's opinion?

Reply:
this is a pattern that we use in all Compound Components. It's done this way to make sure that PropsTable can be generated for the documentation website properly.

Reply:
well, I can get used to it then 🤷